### DISCUSSION TITLE
Have a way to turn off phonemizer, or have a general IPA language

### DISCUSSION THREAD

**Discussion by neurlang** [coqui-ai/TTS#1551] [Ideas]
Hello I have an idea, it might be worth it to specify no phonemizer so that the texts (IPA/other phonetic alphabet) are used directly for training.

```
    use_phonemes=True,
    phoneme_language="multi-lingual",
    phonemizer="ipa_phonemizer",
```

here ipa_phonemizer is my phonemizer, it just replaces g with ɡ, ʤ with dʒ, ʧ with tʃ currently.

This could be useful for languages which do not yet have a way to generate phones from the written texts. So you can experiment with similar-language model more easily and eventually build up your your language -> IPA dict.

**Comment by erogol** [comment]
I don't quite get what you mean. 

If you don't want to use G2P, you can disable it. 
You can enable it and set the language if it is supported by any of the backends we support. 
If you need something different, you can write your own Tokenizer or Phonemizer. 

Do you have something other than these?
